### PR TITLE
#167733640 Setup backend application to use docker

### PR DESCRIPTION
**Pull request by King-Success**
#### Description of Task to be completed?
Should ensure that docker is properly setup on this application
#### How should this be manually tested?
On the project root directory, run `docker-compose up -d` this should start a local development server with all the necessary services running on the background
#### Any background context you want to provide?
None
#### What are the relevant pivotal tracker stories?
#167733640
#### Screenshots (if appropriate)
Screen Shot 2019-08-15 at 1 34 24 AM

### CODE REVIEW

**Review by emmanuelnwankwo — APPROVED** [review]
My first time of using Docker, the installation is straightforward and it's working

**Comment by daylay92** [comment]
Its also my first time using docker and I found the setup to be working fine on my machine. Kindly confirm that you have sorted out the database concern that was mentioned during the standup session today

**Comment by King-Success** [comment]
> Its also my first time using docker and I found the setup to be working fine on my machine. Kindly confirm that you have sorted out the database concern that was mentioned during the standup session today

Thanks @daylay92 for pointing that out. After a bit of research, I figured that the database connection wouldn't be a problem, the database instance running on docker allows you to run the application and connect to the database (Postgres) without you needing to install Postgres database on your local machine. The application would still work exactly the same way as though you are connecting to a database on your local machine.

**Comment by daylay92** [comment]
Thanks for clarifying this.

**Review by daylay92 — APPROVED** [review]

File: src/index.js [hunk 1]
```diff
@@ -18,7 +18,7 @@ const app = express();
 app.use(cors());
 
 // Normal express config defaults
-app.use(require("morgan")("dev"));
+// app.use(require("morgan")("dev"));
```
Comment (efeguono) [inline src/index.js]: Why are these comments here?

**Pull request merged**